src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
```

Comment:
```suggestion
  const calculator = {
```

no need to create new variable for storing object, you can immediately return it,